packages/create-app/index.js

```diff
@@ -152,6 +146,33 @@ function copy(src, dest) {
   }
 }
 
+async function getValidPackageName(projectName) {
+  const packageNameRegExp = /^(?:@[a-z0-9-*~][a-z0-9-*._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/
+  if (packageNameRegExp.test(projectName)) {
+    return projectName
+  } else {
+    const suggestPkgName = projectName
```

Comment:
Everything looks good, the only thing is I think we should rename `suggestPkgName` to `suggestedPackageName`, because we may enable a spellchecker later, and we could merge this. I think we don't need Evan's input for this one.

Reply:
Should we rename `pkg` and `pkgManager` too or it is better done as a separate PR with complete naming convention overhaul?

Reply:
We can do in another PR the overhaul, let's just change this one, so it is the same as `packageNameRegExp` above